_posts/2023-07-10-custom-http-signaling-notification.adoc

```diff
@@ -0,0 +1,252 @@
+
+[source,shell]
+----
+http-signaling-notification-connect-1     | 2023-07-07 15:09:48,205 INFO   Postgres|dbserver1|postgres-connector-task  Recorded signal event 'SignalRecord{id='924e3ff8-2245-43ca-ba77-2af9af02fa07', type='log', data='{"message":"Signal message received from http endpoint."}', additionalData={}}'    [io.debezium.examples.signal.HttpSignalChannel]
+----
+
+Additionally, you might notice log messages related to notification events being sent to the Postbin. For example:
+
+[source,shell]
+----
+http-signaling-notification-connect-1     | 2023-07-07 15:09:47,737 INFO   Postgres|dbserver1|snapshot  [HTTP NOTIFICATION SERVICE] Sending notification to http channel   [io.debezium.examples.notification.HttpNotificationChannel]
+http-signaling-notification-connect-1     | 2023-07-07 15:09:48,347 INFO   Postgres|dbserver1|snapshot  Bin created: {"binId":"1688742588469-1816775151528","now":1688742588470,"expires":1688744388470}   [io.debezium.examples.notification.HttpNotificationChannel]
+----
+
+It provides information about the notification event, such as the creation of a bin with a unique identifier (binId) and other relevant details. To retrieve the notification event from Postbin, fetch the `binId` from the log message and use it to request the corresponding notification event from Postbin. To view the notification event, you can access Postbin using the following URL: `https://www.toptal.com/developers/postbin/b/:binId`. Replace `:binId` in the URL with the actual binId obtained from the connector logs.
+
+The notification event sent to Postbin looks like the following:
+
+++++
+<div class="imageblock centered-image">
+    <img src="/assets/images/http_notification_postbin.png" class="responsive-image" alt="Postbin Preview">
+</div>
+++++
+
+== Conclusion
+
+In this tutorial, we explored how to create custom signal and notification channels for Debezium connectors. We created a custom signal channel that receive a signal event from an HTTP endpoint. We also created a custom notification channel that sends a notification event to an HTTP endpoint.
+
+Debezium's comprehensive signaling and notification system offers seamless integration with third-party solutions, allowing users to stay informed about the state and progress of Debezium connectors. The system's extensibility empowers users to customize both the signals and notification channels to fit their customized needs.
+
+Stay tuned for Part 3 of this series where we will explore about JMX signaling and notifications. In the meantime, you can check out the Debezium documentation for more information about signal and notification channels.
+
+If you have any questions or feedback, please feel free to reach out to us on the Debezium mailing list or the #community-general channel on the Zulip chat. We would love to hear from you!
```

Comment:
Perhaps embed the links to the mailing list and the Zulip channel for quick accessibility.